src/content/docs/analyzer/import-sorting.mdx

```diff
@@ -113,6 +113,40 @@ import "../polyfills/array/flatMap";
 import { functionThatUsesFlatMap } from "./utils.js";
 ```
 
+## Side effect imports
+
+A [side effect imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#forms_of_import_declarations) are import statements that don't import any name:
+
+```js
+import "./global.js"
+```
+
+Since it is hard to determine which side effect the module brought,
+The import organizer creates a group for every side effect imports.
```

Comment:
```suggestion
the import sorting creates a group for every side effect imports.
```

Not sure why you went to a new line after a comma, it breaks the period and it's difficult to read (source code)

Also, **import sorting** 😅

Reply:
A single Markdown line break doesn't translate to a line break? We need a blank line to have a line break no?

Reply:
No. You need two new lines. Check the rendered website. Plus, as I said, going to a new line after a comma is incorrect